docs/src/misc.rst

```diff
@@ -579,7 +579,7 @@ API
 
     .. versionadded:: 1.25.0
 
-.. c:function:: int uv_gettimeofday(uv_timeval_t* tv)
+.. c:function:: int uv_gettimeofday(uv_timeval64_t* tv)
```

Comment:
Should `uv_timeval64_t` be documented under data types at the top of this file? Maybe `uv_timeval_t` as well (it's listed as a type of one of the fields in the `uv_rusage_t` but doesn't appear to be otherwise listed.